lib/index.js

```diff
@@ -162,10 +177,16 @@ api.addPublicKey = (actor, publicKey, privateKey, callback) => {
 
       // insert the publc key
       const now = Date.now();
+
+      // FIXME: this is a sticky wicket, there is a unique compound index
+      // this uses `pem`, how to make this work in a backward compatible way?
```

Comment:
I think removing it would be a fine option as I do believe it was for dup prevention.  However, if we do that, how would an existing database be affected?  Would we need to explicitly delete that index if it exists to provide an upgrade path?

Reply:
@mattcollier,

Yes, we'd have to do that. Do we need backwards compatibility here? We need to push forward with a number of other breaking fixes in related modules and the rest of the updates here are for new software. So I recommend we take the easy path and do a breaking release.